sklearn/semi_supervised/label_propagation.py

```diff
@@ -358,6 +362,18 @@ class distributions will exceed 1 (normalization may be desired).
             affinity_matrix /= normalizer[:, np.newaxis]
         return affinity_matrix
 
+    def fit(self, X, y):
+        # alpha is deprecated for LabelPropagation because it doesn't have any
+        # theoretical meaning (from the reference paper). Look at PR 6727.
+        if self.alpha is not None:
```

Comment:
I don't see how. I have no problem putting a new `__init__` in `LabelPropagation` which passes alpha=None to `BaseLabelPropagation.__init__`, while still passing the tests.